<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeTool</class>
 <widget class="QWidget" name="VolumeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="volumeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>300</height>
    </size>
   </property>
   <widget class="QPushButton" name="silenceBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>35</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeRatio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>35</height>
     </size>
    </property>
    <property name="text">
     <string>45%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="volumeBox" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>220</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>220</height>
     </size>
    </property>
    <widget class="QFrame" name="inLine">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>25</y>
       <width>4</width>
       <height>180</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>4</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>4</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="outLine">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>25</y>
       <width>4</width>
       <height>180</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
